.github/workflows/e2e.yml

```diff
@@ -81,7 +81,7 @@ jobs:
           path: |
             .cache/sqldump.sql
             .env/admin_api_key.txt
-          key: db-cache-${{ hashFiles('pkg/identityserver/store/**/*.go', 'cmd/ttn-lw-stack/commands/is-db.go') }}
+          key: -db-cache-${{ hashFiles('pkg/identityserver/store/**/*.go', 'cmd/ttn-lw-stack/commands/is-db.go') }}
```

Comment:
Why?

Reply:
Please disregard. This was testing only.